packages/app/src/runner/selector-playground/SelectorPlaygroundTooltip.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <Tooltip
+    :triggers="[]"
+    :hide-triggers="[]"
+    :distance="8"
+    hide-arrow
+    :auto-hide="false"
+    :shown="shown"
+    handle-resize
+  >
+    <div
+      :key="'popover-container'"
+      @mouseover="mouseOver"
+      @mouseleave="mouseLeave"
+      @click="click"
+    >
+      <slot />
+    </div>
+    <template #popper>
+      <div
+        class="whitespace-nowrap"
+        data-cy="selector-playground-tooltip"
+      >
+        {{ textToShow }}
+      </div>
+    </template>
+  </Tooltip>
+</template>
+
+<script lang="ts" setup>
+import { ref } from 'vue'
+import { promiseTimeout } from '@vueuse/core'
+import Tooltip from '@packages/frontend-shared/src/components/Tooltip.vue'
+
+const props = defineProps<{
+  hoverText: string
+  clickText?: string
+}>()
+
+const shown = ref(false)
+const textToShow = ref(props.hoverText)
+
+function mouseOver () {
+  shown.value = true
+}
+
+async function mouseLeave () {
+  shown.value = false
+  await promiseTimeout(200)
+  textToShow.value = props.hoverText
```

Comment:
Rather than reseting the value on leave and dealing with the animation timeout, can we just set the value to hoverText before we show it in `mouseOver`?

Reply:
This is so simple! Had to change `mouseover` to `mouseenter` since mouseover triggers on any movement inside the element.